tf-modules/ingest/iam.tf

```diff
@@ -18,6 +18,38 @@ data "aws_iam_policy_document" "states_assume_role_policy" {
   }
 }
 
+# URS password
+# Secrets setup for URS auth in ingest tasks
+
+resource "aws_secretsmanager_secret" "ingest_urs_password" {
+  name_prefix = "${var.prefix}-ingest_urs_password"
+  description = "URS user pasword for use by ingest lambdas in the ${var.prefix} deployment"
+  tags        = var.tags
+}
+
+resource "aws_secretsmanager_secret_version" "ingest_urs_password" {
+  count         = length(var.urs_password) == 0 ? 0 : 1
+  secret_id     = aws_secretsmanager_secret.ingest_urs_password.id
+  secret_string = var.urs_password
+}
+
+data "aws_iam_policy_document" "lambda_processing_role_ingest_urs_password" {
+  statement {
+    actions   = ["secretsmanager:GetSecretValue"]
+    resources = [
+      aws_secretsmanager_secret.ingest_urs_password.arn
+
+    ]
+  }
+}
+
+resource "aws_iam_role_policy" "lambda_processing_role_ingest_urs_password" {
+  role   = split("/", var.lambda_processing_role_arn)[1]
```

Comment:
What is this doing?

Reply:
And why not use var.lambda_processing_role_id?

Reply:
What it's doing is being removed, this is prior work cruft. 

2ade429fb

Reply:
I believe this  became needed again courtesy of  https://github.com/nasa/cumulus/pull/1496/files#r387263029